tn_api_pricing_booking_spec.yml

```diff
@@ -420,6 +420,81 @@ components:
+            type: object
+            description: Hotels
+            properties:
+              hotel_trace_id:
+                type: string
+                example: 'ndsf6sjf6sdf7sdf6'
+                description: An identifier to refer to the hotel for searching room availability
+              status:
+                enum: [booked, cancelled]
+                description: "Status of the hotel booking"
+              booked_hotel_details:
+                type: object
+                description: The specific details of the booked hotel.
+                properties:
+                  address:
+                    type: object
+                    properties:
+                      street_address:
+                        type: string
+                        description: The street address of the hotel.
+                        example: '1583 Brunswick St, Halifax, NS B3J 3P5'
+                      longitude:
+                        type: float
+                        description: The longitude of the hotel.
+                        example: 44.64511447836071
+                      lattitude:
+                        type: float
+                        description: The lattitude of the hotel.
+                        example: -63.575245615440565
+                      distance_from_search_reference:
+                        type: float
+                        description: The distance that the hotel is from the reference submitted in the search.
+                        example: 7.5
+                      phone_number:
+                        type: str
+                        description: The contact phone number for the hotel.
+                        example: '+1 902-420-0555'
+                  hotel_chain:
+                    type: string
+                    description: The refence for the hotel chain.
+                    example: 'GI'
+                  hotel_code:
+                    type: string
+                    description: The reference for the hotel code
+                    example: '35573'
+                  hotel_name:
+                    type: string
+                    description: The name of the hotel
+                    example: 'Cambridge Suites Hotel Halifax'
+                  location_iata:
+                    type: string
+                    description: The location of the hotel.
+                    example: 'LON'
+                  check_in_time:
+                    type: string
+                    description: The check in time for the hotel.
+                    example: "3pm"
+                  check_out_time:
+                    type: string
+                    description: The check out time
+                    example: "10am"
+                  rate_plan_type:
+                    type: string
+                    example: 'A00BOB'
+                    description: The type reference for the rate plan selected for this hotel and this room.
+                  total_price:
```

Comment:
I don't see a currency attached to this - would we potentially have issues where the hotel is booked in GBP if it's in London and therefore we get back 100 but the flights were booked in CAD and therefore we'd need to have 171 CAD or something? How are we dealing with that?

Reply:
Oh i actually need to make a card about that - it returns a random currency from the bookings retrieval. So I need to investigate how the currency is being handled all the way through.